test/translation/transformation.spec.ts

```diff
@@ -12,9 +12,12 @@ const fixtures = fs
     .sort()
     .map(f => [path.parse(f).name, fs.readFileSync(path.join(fixturesPath, f), "utf8")]);
 
-test.each(fixtures)("Transformation (%s)", (_name, content) => {
+test.each(fixtures)("Transformation (%s)", (name, content) => {
     util.testModule(content)
-        .setOptions({ luaLibImport: tstl.LuaLibImportKind.Require })
+        .setOptions({
+            luaLibImport: tstl.LuaLibImportKind.Require,
+            luaTarget: name.includes("luau") ? tstl.LuaTarget.Luau : undefined,
```

Comment:
I was expecting just a second test in this file that hardcodes your file name and luau target, but this works too I guess